src/nvim/main.c

```diff
@@ -1972,6 +1972,35 @@ static bool do_user_initialization(void)
   return do_exrc;
 }
 
+// Read initialization commands from ".nvimrc.lua", ".nvimrc" or ".exrc" in
+// current directory.  This is only done if the 'exrc' option is set.
+// Only do this if VIMRC_FILE is not the same as vimrc file sourced in
+// do_user_initialization.
+static void do_exrc_initialization(void)
+{
+  char *str;
+
+  if (os_path_exists(VIMRC_LUA_FILE)) {
+    str = nlua_read_secure(VIMRC_LUA_FILE);
+    if (str != NULL) {
+      nlua_exec(cstr_as_string(str), (Array)ARRAY_DICT_INIT, &ERROR_INIT);
```

Comment:
How do we handle the error here? 🤔

Reply:
```suggestion
      Error err;
      nlua_exec(cstr_as_string(str), (Array)ARRAY_DICT_INIT, &err);
      if (ERROR_SET(&err)) {
        // ...
        api_clear_error(&err);
      }
```

Reply:
Updated.

Now for this `.nvimrc.lua` file:

```lua
print('nvimrc.lua')

require('invalid.module.name')
```

the result is:

![image](https://user-images.githubusercontent.com/8050659/208095149-d01253c1-8a2c-4e2e-9f0f-b123183d9910.png)

Reply:
For comparison,

With this `.nvimrc`:

```vim
echo "nvimrc"

invalid "module-name"
```

the result is:

![image](https://user-images.githubusercontent.com/8050659/208095346-3aaac498-f301-4592-bb1c-0b1b5e147773.png)

Reply:
They are different errors, the fact that one is longer than the other is not an issue.

Reply:
Ah, I wasn't showing that they are different. I was just showing that I kept the first line similar. 😂